src/main/java/net/minecraftforge/client/event/GuiScreenEvent.java

```diff
@@ -159,6 +159,20 @@ public int getMouseY()
             return mouseY;
         }
     }
+
+    /**
+     * This event fires in {@code InventoryEffectRenderer.updateActivePotionEffects()}
```

Comment:
I don't know much about javadoc syntax, I just copied the style I saw elsewhere in the file. Might be worth fixing up all the javadocs in another PR?